src/main/java/seedu/cafectrl/storage/Storage.java

```diff
@@ -35,6 +36,14 @@ public Storage (Ui ui) {
     }
 
     //@@author Cazh1
+
+    private boolean isFileEmpty(ArrayList<String> encodedStringArrayList) {
+        if (encodedStringArrayList.isEmpty()) {
+            return true;
+        }
+        return false;
```

Comment:
Would it be better to write this as a single line?
```suggestion
        return encodedStringArrayList.isEmpty();
```